frontend/public/components/default-resource.tsx

```diff
@@ -64,6 +67,9 @@ export const DetailsForKind = (kind: string) =>
                 showNodeSelector={false}
               />
             </div>
+            {additionalDetailsCallback && (
+              <div className="col-md-6">{additionalDetailsCallback(obj)}</div>
```

Comment:
If `additionalDetailsCallback` return null then an empty div will rendered.

Reply:
added check to restrict render when function returns null or undefined.